gripper/gripper.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+
+import argparse
+import random
+
+parser = argparse.ArgumentParser()
+
+parser.add_argument("balls", type=int, help="the number of balls")
+parser.add_argument("--seed", "-s", type=int, help="random seed")
+parser.add_argument("--randomize", "-r", action="store_true", help="randomize the initial state and the goal if present")
```

Comment:
I think it's better to combine these two options into the `--seed` option and randomize if `--seed` is given.